src/css/docs.css

```
@@ -0,0 +1,1462 @@
+div.docs section#hero:before {
+	content: "";
+	display: none;
+}
+
+div.docs section#hero header {
+	background-color: #263345;
+	position: relative;
+	z-index: 100;
+	padding: 0;
+}
+
+div.docs header nav {
+	height: 100%;
+}
+
+div.docs header .wrapper {
+	height: 90px;
+	align-items: center;
+}
+
+div.docs header .wrapper nav {
+	margin: 0;
+	display: flex;
+}
+
+div.docs div.api-category .wrapper.link-back {
+	transition: 0.2s ease all;
+	text-decoration: none;
+	position: relative;
+	padding: 15px 40px;
+	margin: 15px auto;
+	z-index: 101;
+}
+
+div.docs div.api-category .wrapper.link-back:hover {
+	background-color: rgba(255, 255, 255, 0.1);
+	border-radius: 3px;
+}
+
+div.docs .menu-link {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	box-sizing: border-box;
+	position: relative;
+	padding-top: 23px;
+}
+
+div.docs .menu-link.active {
+	height: 100%;
+	border-radius: 0px;
+}
+
+div.docs #logo img {
+	width: 161px;
+	margin-top: 4px;
+}
+
+div.docs .menu-link.active:before {
+	content: "";
+	display: block;
+	position: absolute;
+	bottom: -1px;
+	left: 0;
+	height: 2px;
+	width: 100%;
+	background-color: rgba(255, 255, 255, 0.2);
+}
+
+div.docs div.api-category {
+	position: relative;
+	border-top: 1px solid rgba(255, 255, 255, 0.1);
+}
+
+div.docs div.api-category .wrapper {
+	display: flex;
+	align-items: center;
+	padding: 30px 0px;
+}
+
+div.docs div.api-category .category-icon {
+	height: 80px;
+	display: flex;
+	width: 80px;
+	justify-content: center;
+	align-items: center;
+	border-radius: 3px;
+	background-color: rgba(0, 0, 0, 0.1);
+}
+
+div.docs div.api-category .category-icon img {
+	width: 40px;
+	height: 40px;
+}
+
+div.docs div.api-category .category-title {
+	font-family: "Circular", sans-serif;
+	font-size: 36px;
+	color: #FFFFFF;
+	letter-spacing: -0.6px;
+	font-weight: 300;
+	margin: 0 0 -5px 30px;
+}
+
+div.docs section#hero {
+	height: auto;
+	padding-bottom: 0;
+}
+
+div.docs .category-navigation {
+	background-color: #fff;
+	padding: 2px 0 0px 0;
+	border: 1px solid #F0F0F2;
+}
+
+div.docs .category-navigation nav {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+div.docs .category-navigation nav a {
+	font-family: "Circular", sans-serif;
+	font-size: 13px;
+	font-weight: 500;
+	color: #BABABA;
```

`#BABABA` text on a white background has a contrast ratio of 1.94:1 and does not satisfy the Web Content Accessibillity Guidelines (WCAG 2.1), which requires at least a 3:1 contrast for larger text. `#959595` gives us a 3:1 contrast ratio and makes the menu much more readable (in my opinion).

```suggestion
	color: #959595;
```

This needs to be changed in multiple places.